crates/swc_html_codegen/tests/fixture.rs

```diff
@@ -321,6 +321,41 @@ impl VisitMut for DropSpan {
     }
 }
 
+struct NormalizeTest;
+
+impl VisitMut for NormalizeTest {
+    fn visit_mut_document_fragment(&mut self, n: &mut DocumentFragment) {
+        n.visit_mut_children_with(self);
+
+        if let Some(Child::Text(_)) = n.children.last_mut() {
+            // Drop value from the last `Text` node because characters after `</body>`
+            // moved to body tag
+            n.children.remove(n.children.len() - 1);
+        }
+    }
+
+    fn visit_mut_element(&mut self, n: &mut Element) {
+        n.visit_mut_children_with(self);
+
+        if &*n.tag_name == "body" {
+            if let Some(Child::Text(text)) = n.children.last_mut() {
+                // Drop value from the last `Text` node because characters after `</body>`
+                // moved to body tag
+                text.value = "".into();
+            } else {
+                n.children.push(Child::Text(Text {
+                    span: Default::default(),
+                    value: "".into(),
```

Comment:
Same here

Reply:
@kdy1 Rebase problem, `NormalizeTest` no need anymore and should be removed :smile: